code/modules/projectiles/guns/smgs.dm

```diff
@@ -324,10 +324,11 @@
 	current_mag = /obj/item/ammo_magazine/smg/uzi
 	attachable_offset = list("muzzle_x" = 30, "muzzle_y" = 20,"rail_x" = 11, "rail_y" = 22, "under_x" = 22, "under_y" = 16, "stock_x" = 22, "stock_y" = 16)
 
-	fire_delay = 0.175 SECONDS
+	fire_delay = 0.15 SECONDS
 	burst_amount = 4
-	accuracy_mult_unwielded = 0.85
-	scatter = 15
-	scatter_unwielded = 60
+	accuracy_mult_unwielded = 0.9
+	accuracy_mult = 1
+	scatter = 0
+	scatter_unwielded = 10
```

Comment:
```suggestion
	scatter = 10
	scatter_unwielded = 15
```

Reply:
the uzi has nothing over a machine pistol other than 2 more bullets and better akimbo stats, don't see why it needs 10-15 scatter